Lib/tkinter/__init__.py

```diff
@@ -4877,26 +4893,17 @@ def sash_place(self, index, x, y):
         return self.sash("place", index, x, y)
 
     def panecget(self, child, option):
-        """Query a management option for window.
-
-        Option may be any value allowed by the paneconfigure subcommand
-        """
+        """Return the value of option for window."""
         return self.tk.call(
             (self._w, 'panecget') + (child, '-'+option))
 
     def paneconfigure(self, tagOrId, cnf=None, **kw):
-        """Query or modify the management options for window.
-
-        If no option is specified, returns a list describing all
-        of the available options for pathName.  If option is
-        specified with no value, then the command returns a list
-        describing the one named option (this list will be identical
-        to the corresponding sublist of the value returned if no
-        option is specified). If one or more option-value pairs are
-        specified, then the command modifies the given widget
-        option(s) to have the given value(s); in this case the
-        command returns an empty string. The following options
-        are supported:
+        """Query or modify the configuration options for window TAGORID.
```

Comment:
It took me a while to understand what `TAGORID` meant, maybe it'd be better if we match the parameter name, i.e. use `tarOrId` (and similar for the other cases)?

Reply:
As I indicated elsewhere, I think the raw test should be `*tagOrID*` as italicizing option names is better than all-capping them.

Reply:
In THIS file, capitalization is used for parameter names. It is not ideal, but we should be consistent.